sci-biology/abyss/abyss-2.0.2-r1.ebuild

```diff
@@ -47,9 +52,19 @@ src_prepare() {
 }
 
 src_configure() {
+	local myconf=""
 	# disable building haskell tool Misc/samtobreak
 	# unless request by user: bug #534412
 	use misc-haskell || export ac_cv_prog_ac_ct_GHC=
 
-	econf $(use_enable openmp)
+	if ! use sqlite; then
+		myconf="${myconf} --without-sqlite"
```

Comment:
Can you use $(use_with) instead of `if ... fi`?

Reply:
@trofi Done. The boost DEPEND line could maybe be "improved" like in dev-python/graph-tool-2.23.ebuild ? Well, I do not know what that really does.

        `dev-libs/boost:=[context,python,${PYTHON_USEDEP}]`